app/views/idv/in_person/location.html.erb

```diff
@@ -0,0 +1,15 @@
+<% title t('titles.doc_auth.verify') %>
+
+<h1 class="h3 margin-y-0">
+  <%= t('in_person_proofing.headings.location') %>
+</h1>
+
+<%= validated_form_for :doc_auth,
+                       url: url_for,
+                       method: 'put',
+                       html: { autocomplete: 'off', class: 'margin-y-5 js-consent-continue-form' } do |f| %>
```

Comment:
I think the `js-consent-continue-form` class is only needed for the welcome step, so let's remove it from all these other steps?

https://github.com/18F/identity-idp/blob/5e0e9b2ad22637e6f275b0b8bb1b4b82293aa031/app/javascript/packs/document-capture-welcome.js#L3

```suggestion
                       html: { autocomplete: 'off', class: 'margin-y-5' } do |f| %>
```

Reply:
fixed in 77668529e3e40a2e233d4617f6b3e1bccab2d042